From 9f9ee0f46310cbd92a91b7ae2f3dd0078789bb14 Mon Sep 17 00:00:00 2001
From: Nishanth Menon <nm@ti.com>
Date: Mon, 14 Feb 2011 16:47:23 +0530
Subject: [PATCH 31/47] omap3630: hwmod: sr: enable for higher ES as well

Enable hwmod entries for OMAP3630 for higher ES revisions as
well

Signed-off-by: Nishanth Menon <nm@ti.com>
---
 arch/arm/mach-omap2/omap_hwmod_3xxx_data.c |    8 ++++++--
 1 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-omap2/omap_hwmod_3xxx_data.c b/arch/arm/mach-omap2/omap_hwmod_3xxx_data.c
index c629d6c..aa0e7e0 100644
--- a/arch/arm/mach-omap2/omap_hwmod_3xxx_data.c
+++ b/arch/arm/mach-omap2/omap_hwmod_3xxx_data.c
@@ -1321,7 +1321,9 @@ static struct omap_hwmod omap36xx_sr1_hwmod = {
 	},
 	.slaves		= omap3_sr1_slaves,
 	.slaves_cnt	= ARRAY_SIZE(omap3_sr1_slaves),
-	.omap_chip	= OMAP_CHIP_INIT(CHIP_IS_OMAP3630ES1),
+	.omap_chip	= OMAP_CHIP_INIT(CHIP_IS_OMAP3630ES1	|
+					CHIP_IS_OMAP3630ES1_1	|
+					CHIP_IS_OMAP3630ES1_2),
 	.mpu_irqs	= omap3_smartreflex_mpu_irqs,
 	.mpu_irqs_cnt	= ARRAY_SIZE(omap3_smartreflex_mpu_irqs),
 };
@@ -1371,7 +1373,9 @@ static struct omap_hwmod omap36xx_sr2_hwmod = {
 	},
 	.slaves		= omap3_sr2_slaves,
 	.slaves_cnt	= ARRAY_SIZE(omap3_sr2_slaves),
-	.omap_chip	= OMAP_CHIP_INIT(CHIP_IS_OMAP3630ES1),
+	.omap_chip	= OMAP_CHIP_INIT(CHIP_IS_OMAP3630ES1	|
+					CHIP_IS_OMAP3630ES1_1	|
+					CHIP_IS_OMAP3630ES1_2),
 	.mpu_irqs	= omap3_smartreflex_core_irqs,
 	.mpu_irqs_cnt	= ARRAY_SIZE(omap3_smartreflex_core_irqs),
 };
-- 
1.7.1

